### ISSUE TITLE
Move all script to one project oxExternal

### ISSUE THREAD

**Issue by yurem** [GluuFederation/community-edition-setup#683]
Currently we store custom scripts in few different places (projects):
1) Setup
2) oxAuth
3) oxExternal
4) Other projects

As result we have to sync scripts between all places. This can led to issues.

I offer to move all scripts to oxExternal (or rename this projects). We can add to this project maven build to make zip and deploy it to maven repo.

After that we can use this zip in setup.

**Comment by yurem** [comment]
@nynymike @yuriyz What do you think to moving all scripts to oxExternal?

**Comment by yuriyz** [comment]
Sounds good to me.

**Comment by mbaser** [comment]
I think we can easiliy implement this when we move to mono-repo

**Comment by yuriyz** [comment]
Right, lets keep it on hold since main structure is going to change.

**Comment by jgomer2001** [comment]
should we close this one?
